pkg/apis/pmemcsi/v1beta1/deployment_conversion.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/conversion"
+)
+
+var _ conversion.Hub = &Deployment{}
+
+// Hub marks this(v1beta1) type as a conversion hub.
+func (*Deployment) Hub() {}
```

Comment:
Explaining "hub" as "conversion hub" doesn't explain what a "hub" is (circular definition). Please explain what it does.

Reply:
This is copied from the generated code by controller-runtime/operator-sdk. Added a short description:
`// Conversion hub is the version that all other CRD versions get converted into.`